SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -3,12 +3,13 @@ import Cocoa
 // 10 MiB.
 let kMSDefaultDatabaseSize = 10 * 1024 * 1024
 
-class AppCenterViewController : NSViewController, NSTextFieldDelegate {
+class AppCenterViewController : NSViewController, NSTextFieldDelegate, NSTextViewDelegate {
 
   var appCenter: AppCenterDelegate = AppCenterProvider.shared().appCenter!
   var currentAction = AuthenticationViewController.AuthAction.signin
 
   let kMSAppCenterBundleIdentifier = "com.microsoft.appcenter";
+  let prodLogUrl = "https://in.appcenter.ms"
```

Comment:
You can use empty value to reset, and then get the current URL from SDK to display.

Reply:
Are you talking about exposing logUrl in `MSAppCenter` class? There is only setter available as a public API.

Reply:
Yeah, you are right. This is not a puppet, and this API not available (but due to objective c specifics it can be called 😉).
I just wanted to avoid having this constant here.

Reply:
I use "nil" to set value in UserDefaults when reset, and if restarting app, it will skip setLogUrl call and continue AppCenter start. So the SDK will still use default log url (PROD URL: https://in.appcenter.ms ).
For the UI display, if it don't need to display PROD Log URL,  I use text "Log URL currently set to the production"  to point out current log url is PROD url.  Do you think this is ok?

I pushed a new commit for above, and fix "Set Log URL" button display issue, please help to check.